### PR TITLE
Added the About Page

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
Part of #11 

Current About section: 

image

The About Page

image

image

image

CC: @SaptakS @ananyo2012 @vipulgupta2048

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+PyCon India is the largest annual gathering of Python aficionados and experts in India where Pythonistas come from around the world to celebrate their use of Python, and share their knowledge and experience. It is a volunteer-driven conference aimed to foster the adoption, use, and development of the Python programming language. The 12th edition of PyCon was supposed to take place in the beautiful city of Bangalore. But due to the current situation, the conference will take place virtually from 2nd October to 5th October 2020. With this move, this year, we are making the conference bigger, better, and more accessible to Pythonistas all over the world.
+
+## What happens during the conference?
+
+
+PyCon India 2020 consists of 3 events rolled into one mega event for 3 days of learning and fun:
+
+* __Conference Days:__ {Dates here} will be the Conference Days. On the conference days, people will be able to connect with other Pythonistas, discuss their experiences, present their ideas, and attend the talks.
```
Comment (Sc0rpi0n101) [inline about.md]: I would need specific dates for the different events at the conference.

File: about.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+* __Workshop Day:__ Workshop Day will be on {Date}, 2020 (Please note that Workshop overlaps with Devsprints). On the workshop day, people can attend hands-on workshops, and learn about a topic they have been excited about, but didn't get the time or resources to learn about, from an expert.
```
Comment (Sc0rpi0n101) [inline about.md]: Also here

File: about.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+* __Developer Sprints:__ The Developer Sprints will be on {Dates}, 2020. During the Devsprints, people can contribute to open source projects, guided by mentors having experience with open-source and their projects. This is a great opportunity for people to start contributing to Python and other open-source projects. You can learn more about Devsprints from [our blog](https://in.pycon.org/blog/2019/understanding-devsprints.html).
```
Comment (Sc0rpi0n101) [inline about.md]: And here

**Comment by vipulgupta2048** [comment]
@Sc0rpi0n101 Aren't dates on https://in.pycon.org/ not what you are looking for here? Those dates are infact the final dates.

**Comment by Sc0rpi0n101** [comment]
Alright, updated. Any other comments about the content?

File: about.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Want to submit a talk/workshop for the conference?
+
+With PyCon India 2020 being online this year, we are introducing preview videos to our proposal submission workflow. Preview videos are you talking about your proposal, topics you intend to cover in the talk, and how you intend to cover them. Participants are strongly suggested to upload links of their preview videos while submitting their proposals. It will be immensely helpful for reviewers to go through preview videos and take that into account before making final decisions on your talk proposal. You can check out our [CFP page](https://in.pycon.org/cfp/2020/proposals/) and [blog](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html) for more details.
+
+## Who are the Organizers and what is our vision?
+
+### <span style="color:#F6B700">Organized By the people, For the people</span>
```
Comment (vipulgupta2048) [inline about.md]:
```suggestion
### <span style="color:#F6B700">Organized by the people, for the people</span>

```

File: about.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+PyCon India is run every year by Python users and community members all across India to bring their experience and expertise together to share with everyone. The conference is organized entirely by volunteers all the way from planning it, finding the speakers, managing the venue, to completing the event successfully. We have a large team of very enthusiastic and experienced volunteers coming from various Python communities all across India.
```
Comment (vipulgupta2048) [inline about.md]:
```suggestion
PyCon India is organized every year by Python communities and user groups all across India as a way to pool their experience and expertise into making a more successful, more accessible conference every year. The conference is organized entirely by volunteers all the way from planning, to finding the speakers, to managing the event successfully. We have a very large distributed team of enthusiastic and experienced volunteers coming from various Python communities all across India. 

```

**Review by vipulgupta2048 — CHANGES_REQUESTED** [review]
Read it, looks solid, needs a bit of sharpening but I feel its more important to confirm we are giving the right information about our current direction. Need more eyes on this folks, this will be a critical point of information that we are writing. Do give it a read before getting this merged. 
@sayanchowdhury @palnabarun @anistark @satyaakam @SaptakS Thanks!

**Comment by satyaakam** [comment]
> 2009 was the first year and yes, always been a volunteer-driven conference.

Yes its always been  like this , if you see the https://in.pycon.org/2018/ , we depicted with a Graphic if you click on the Year you  would see a picture from that year

**Comment by satyaakam** [comment]
> Ahh, Merriam webster says otherwise https://www.merriam-webster.com/dictionary/cumulative

haaah then we have different spellings from Americans and British as well ;-)

**Comment by ananyo2012** [comment]
@Sc0rpi0n101 Please rebase you changes over the latest master. The preview issue is fixed.

**Comment by Sc0rpi0n101** [comment]
> @Sc0rpi0n101 Please rebase you changes over the latest master. The preview issue is fixed.

Done

**Review by ananyo2012 — APPROVED** [review]

File: _includes/about.html [hunk 6]
```diff
@@ -10,5 +10,10 @@
       across the world who can watch, participate and share their views
       right from the comfort of their homes.
     </div>
+    <div class="content">
+      <a href="/2020/about" class="primary-button">
```
Comment (palnabarun) [inline _includes/about.html]: Can you please make it a relative URL?

Ref: https://talk.jekyllrb.com/t/relative-url-and-baseurl/2051/6

File: about.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: About Us
+---
+
+#  <span style="color:#EB090D">Divided by COVID. Connected by Python</span>
+
+## What is PyCon? 
+
+
```
Comment (palnabarun) [inline about.md:12]: nit: any reason for the double empty lines?

File: about.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+#  <span style="color:#EB090D">Divided by COVID. Connected by Python</span>
+
+## What is PyCon? 
+
+
+PyCon India is the largest annual gathering of Python aficionados and experts in India where Pythonistas come from around the world to celebrate their use of Python, and share their knowledge and experience. It is a volunteer-driven conference aimed to foster the adoption, use, and development of the Python programming language. The 12th edition of PyCon was supposed to take place in the beautiful city of Bangalore. But due to the current situation, the conference will take place virtually from 2nd October to 5th October 2020. With this move, this year, we are making the conference bigger, better, and more accessible to Pythonistas all over the world.
+
+## What happens during the conference?
+
+
```
Comment (palnabarun) [inline about.md:17]: nit: same as above.

File: about.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: About Us
+---
+
+#  <span style="color:#EB090D">Divided by COVID. Connected by Python</span>
+
+## What is PyCon?
```
Comment (palnabarun) [inline about.md]:
```suggestion
## What is PyCon India? 

```

File: about.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: About Us
+---
+
+#  <span style="color:#EB090D">Divided by COVID. Connected by Python</span>
```
Comment (palnabarun) [inline about.md]: Just a thought.

```suggestion
#  <span style="color:#EB090D">Divided by a pandemic. Connected by Python</span>

```

File: about.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+## Want to submit a talk/workshop for the conference?
+
+With PyCon India 2020 being online this year, we are introducing preview videos to our proposal submission workflow. Preview videos are you talking about your proposal, topics you intend to cover in the talk, and how you intend to cover them. Participants are strongly suggested to upload links of their preview videos while submitting their proposals. It will be immensely helpful for reviewers to go through preview videos and take that into account before making final decisions on your talk proposal. You can check out our [CFP page](https://in.pycon.org/cfp/2020/proposals/) and [blog](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html) for more details.
+
+## Who are the Organizers and what is our vision?
```
Comment (palnabarun) [inline about.md]:
```suggestion
## Who are the organizers and what is our vision?

```

File: about.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+### <span style="color:#F6B700">Organized by the people, for the people</span>
+
+
```
Comment (palnabarun) [inline about.md:35]: nit: double empty lines.

File: about.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+PyCon India is organized every year by Python communities and user groups all across India as a way to pool their experience and expertise into making a more successful, more accessible conference every year. The conference is organized entirely by volunteers all the way from planning, to finding the speakers, to managing the event successfully. We have a very large distributed team of enthusiastic and experienced volunteers coming from various Python communities all across India. 
+
+
```
Comment (palnabarun) [inline about.md:38]: nit: double empty lines.

File: about.md [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+Our Vision is to bring the cumulative knowledge and experience of our community members to the public. We want to create a responsible and inclusive ecosystem where Pythonistas from diverse backgrounds can come together to share their stories and gain something from other's experiences. 
+
+The python programming language has become an essential tool for people from various backgrounds and careers, including programmers, mathematicians, doctors, scientists, security researchers, hardware designers, and many more. We aim to ensure the conference can reflect the diversity of our community and facilitate the exchange of ideas, knowledge, and experiences enriching the community.
```
Comment (palnabarun) [inline about.md]:
```suggestion
The Python programming language has become an essential tool for people from various backgrounds and careers, including programmers, mathematicians, doctors, scientists, security researchers, hardware designers, and many more. We aim to ensure the conference can reflect the diversity of our community and facilitate the exchange of ideas, knowledge, and experiences enriching the community. 

```

**Review by palnabarun — CHANGES_REQUESTED** [review]
@Sc0rpi0n101 Pretty solid content. :rocket: 

Other than the reviews, LGTM!

File: _includes/about.html (hunk 6 above)
Comment (Sc0rpi0n101) [reply _includes/about.html]: Sure

File: about.md (hunk 7 above)
Comment (Sc0rpi0n101) [reply about.md:12]: yeah, as you can see in https://in.pycon.org/blog/2020/2020-call-for-volunteers.html, I feel there's not enough space between the headings and the content. So, that's why I've used double lines for the headings.

File: about.md (hunk 10 above)
Comment (Sc0rpi0n101) [reply about.md]: Sound better tbh

File: about.md (hunk 7 above)
Comment (Sc0rpi0n101) [reply about.md:12]: So, should I keep the double lines, or make them single line?

Comment (palnabarun) [reply about.md:12]: Yeah. That makes sense.

Overall, I feel the paragraph spacing needs to be fixed. Although this is an unrelated change.

Comment (palnabarun) [reply about.md:12]: @Sc0rpi0n101 Please keep them for now. We will review later.

File: _includes/about.html (hunk 6 above)
Comment (Sc0rpi0n101) [reply _includes/about.html]: Done

**Review by sayanchowdhury — APPROVED** [review]

File: _includes/about.html [hunk 15]
```diff
@@ -10,5 +10,10 @@
       across the world who can watch, participate and share their views
       right from the comfort of their homes.
     </div>
+    <div class="content">
+      <a href="{{ 'about.html' | relative_url }}" class="primary-button">
```
Comment (palnabarun) [inline _includes/about.html]: Something I missed reviewing earlier:

```suggestion
      <a href="{{ '/about' | relative_url }}" class="primary-button">
```

The `Learn More` link is going to a `.html` page which doesn't look much good URL wise.

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: _includes/about.html (hunk 15 above)
Comment (Sc0rpi0n101) [reply _includes/about.html]: I was going to do just about. But, I saw the coc page was .html, so I added it.

**Review by palnabarun — APPROVED** [review]

**Comment by palnabarun** [comment]
@Sc0rpi0n101 -- Thank you for all your efforts with this. :+1:

**Comment by sayanchowdhury** [comment]
Yay :clinking_glasses:

**Pull request merged**